docker/Dockerfile.release

```
@@ -0,0 +1,98 @@
+# Download artifact from bintray, where the artifact should have been published by now.
+
+FROM openzipkin/zipkin-builder as built
+
+ARG version
+
+# Download jars using Maven. It will try to resolve the artifact from Maven Central, which might
+# not work right away if the sync is taking time, followed by bintray, which should always work
+# since it's where we publish to.
+
+RUN mvn org.apache.maven.plugins:maven-dependency-plugin:get \
+  -DremoteRepositories=bintray::::http://dl.bintray.com/openzipkin/maven -Dtransitive=false \
+  -Dartifact=io.zipkin:zipkin-server:${version}:jar:exec
+RUN mvn org.apache.maven.plugins:maven-dependency-plugin:get \
+  -DremoteRepositories=bintray::::http://dl.bintray.com/openzipkin/maven -Dtransitive=false \
+  -Dartifact=io.zipkin:zipkin-server:${version}:jar:slim
+
+# Extract the built out of the Maven repository
+
+RUN mkdir -p /zipkin && cp ~/.m2/repository/io/zipkin/zipkin-server/${version}/zipkin-server-${version}-exec.jar /zipkin && cd /zipkin && jar xf *.jar && rm *.jar
+RUN mkdir -p /zipkin-slim && cp ~/.m2/repository/io/zipkin/zipkin-server/${version}/zipkin-server-${version}-slim.jar /zipkin-slim && cd /zipkin-slim && jar xf *.jar && rm *.jar
+
+# Extract zipkin-lens
+RUN mkdir -p /zipkin-lens && cp /zipkin/BOOT-INF/lib/zipkin-lens-${version}.jar /zipkin-lens && cd /zipkin-lens && jar xf *.jar && rm *.jar
+
+#####
+# zipkin-ui - An image containing the Zipkin web frontend only, served by NGINX
+#####
+
+FROM nginx:1.16-alpine as zipkin-ui
+LABEL MAINTAINER Zipkin "https://zipkin.io/"
+
+ENV ZIPKIN_BASE_URL=http://zipkin:9411
+
+COPY --from=built /zipkin-lens /var/www/html/zipkin
+RUN mkdir -p /var/tmp/nginx && chown -R nginx:nginx /var/tmp/nginx
+#
+# Setup services
+COPY docker/lens/nginx.conf /etc/nginx/conf.d/zipkin.conf.template
+COPY docker/lens/run.sh /usr/local/bin/nginx.sh
+
+EXPOSE 80
+
+CMD ["/usr/local/bin/nginx.sh"]
+
+#####
+# zipkin-slim - An image containing the slim distribution of Zipkin server.
+#####
+
+FROM openzipkin/jre-full:11.0.4-11.33 as zipkin-slim
+LABEL MAINTAINER Zipkin "https://zipkin.io/"
+
+# Use to set heap, trust store or other system properties.
+ENV JAVA_OPTS -Djava.security.egd=file:/dev/./urandom
+# 3rd party modules like zipkin-aws will apply profile settings with this
+ENV MODULE_OPTS=
+
+RUN adduser -g '' -h /zipkin -D zipkin
+
+# Add environment settings for supported storage types
+COPY --from=built --chown=zipkin /zipkin-slim/ /zipkin/
+COPY --chown=zipkin docker/zipkin/ /zipkin/
+WORKDIR /zipkin
+
+RUN ln -s /busybox/* /bin
+
+USER zipkin
+
+EXPOSE 9410 9411
```

I think we actually removed thrift from slim

```suggestion
EXPOSE 9411
```